### PR TITLE
Update the CBMC submodule

### PR DESCRIPTION
**Pull request by martin-cs**

### CODE REVIEW

**Review by chrisr-diffblue — APPROVED** [review]
Approving - but I'm also doing so "at risk" as we currently do not have working CI so I can't be sure this won't break once merged. I'll leave it up to the author to take the responsibility of hitting the merge button :-)

**Comment by martin-cs** [comment]
Thanks for the review.  I have checked this elsewhere but I may wait pressing merge until the new CI set up is ready!

**Comment by martin-cs** [comment]
CI is now working and passes!

**Pull request merged**
